zetaclient/evm_signer.go

```diff
@@ -318,7 +318,7 @@ func (signer *EVMSigner) SignCommandTx(
 }
 
 func (signer *EVMSigner) TryProcessOutTx(
-	send *types.CrossChainTx,
+	crossChainTx *types.CrossChainTx,
```

Comment:
Good to rename those. I think it is fine to use `cctx` instead of `crossChainTx` since it's a term we use everywhere in the codebase

Reply:
ec6b0d8eb62f0f456266e7dc6dd7668ea0612f75